src/praisonai/praisonai/__main__.py

```
@@ -263,6 +278,51 @@ def _get_typer_commands():
+        return _legacy_verbs_cache
+
+    with _legacy_verbs_lock:
+        if _legacy_verbs_cache is not None:  # Double-check
+            return _legacy_verbs_cache
+
+        try:
+            from praisonai.cli.legacy.dispatch.argparse_builder import (
+                LEGACY_SPECIAL_COMMANDS,
+            )
+            verbs = frozenset(LEGACY_SPECIAL_COMMANDS)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)